src/reducers/dependencies.reducer.test.js

```diff
@@ -56,6 +56,7 @@ Object {
       "description": "",
       "homepage": "",
       "license": "",
+      "location": "dependencies",
```

Comment:
Thanks for fixing this! We really need to get CI up so that we can trust we aren't merging anything in that breaks the tests.